apps/portals/src/configurations/elportal/synapseConfigs/projects.ts

```diff
@@ -49,9 +49,9 @@ export const projectsDetailsPageConfiguration: DetailsPageProps = {
   synapseConfigArray: [
     {
       name: 'CardContainerLogic',
-      columnName: 'grant',
+      columnName: 'shortName',
       title: 'Studies',
-      tableSqlKeys: ['grant'],
+      tableSqlKeys: ['projectShortName'],
```

Comment:
Use the `shortName` column to find those studies that match on the `projectShortName` column